src/cuter_cerl.erl

```diff
@@ -70,7 +70,7 @@
 -type name() :: atom().
 -type fa() :: {name(), arity()}.
 -type cerl_attr_type() :: cerl_recdef() | cerl_typedef().
--type cerl_attr_spec() :: cerl_specdef().
+-type cerl_attr_spec() :: cerl_spec_form().
```

Comment:
Motivated by this change, a more general comment:
Currently, we have three different type names for the same thing in this module (`cerl_attr_spec`, `cerl_spec_form` and `spec_info` a bit further down below) and moreover two of these type names are exported.   Perhaps we should choose one of them and stick to it.